DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommandLineTool/AnonymizerOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using CommandLine;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Tool
```

Comment:
The namespaces for the tool project is a little bit weird. My understanding is that `Tool` does not belongs to `Core`?